scripts/operations/node_management/ncn-image-modification.sh

```diff
@@ -423,6 +423,47 @@ function create_new_squashfs() {
   done
 }
 
+function csm_15X_gpg_patch {
+  local gpg_keys=()
+  local script_path
+  local scripts_path
+  local gpg_keys_path
+
+  script_path="$(rpm -ql docs-csm | grep "$(basename "$0")")"
+  scripts_path="$(dirname "$script_path")"
+  gpg_keys_path="${scripts_path}/keys"
```

Comment:
This way `gpg_keys_path` will be evaluated to `/usr/share/doc/csm/scripts/operations/node_management/keys`. It's actually one level up (`/usr/share/doc/csm/scripts/operations/keys/`).